lib/experimental/interactivity-api/modules.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Interactive modules.
+ *
+ * @package Gutenberg
+ * @subpackage Interactivity API
+ */
+
+/**
+ * Register the `@wordpress/interactivity` module.
+ */
+function gutenberg_register_interactivity_module() {
+	gutenberg_register_module(
+		'@wordpress/interactivity',
+		'/wp-content/plugins/gutenberg/build/interactivity/index.min.js',
```

Comment:
Here in other places to make the path work with different WP configurations:
```suggestion
		gutenberg_url( 'build/interactivity/index.min.js' ),
```

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/56143/commits/5488eab4d7021e6dfcb20f0e9685555b8ef09016. Thanks, Greg.